bin/cli.js

```diff
@@ -355,4 +359,34 @@ program
     }
   });
 
-program.parse();
+// Update the CLI
+if (pkg.repository && pkg.repository.url) {
+  program
+    .command("update")
+    .description("Update the CLI to the latest version")
+    .action(() => {
+      console.log(`Updating npm package from GitHub repository...`);
+      exec(
```

Comment:
Can be a long running command; better to also add the spinner here to give the user feedback the process is running?

Reply:
I tried to add the spinner, but it was conflicting with the user input required to be typed into the terminal (password). I'll skip it for now